### ISSUE TITLE
ugly error if user is not not logged in to OpenShift cluster

### ISSUE THREAD

**Issue by kadel** [redhat-developer/ocdev#125]
```
▶ ./ocdev component get
unable to get current component: unable to get active application: unable to get active application: unable to get current project info: command: [/usr/local/bin/oc project] failed to run:
error: You must be logged in to the server (the server has asked for the client to provide credentials (get projects.project.openshift.io myproject))
: exit status 1
```

this should be caught by some pre-flight check and nice error message should be shown

**Comment by containscafeine** [comment]
@kadel can you confirm this was fixed in #99?

**Comment by kadel** [comment]
no it wasn't. I've opened this after #99 was merged.

You get this error, when OpenShift is running, your `oc` credentials expired.
